Cargo.toml

```diff
@@ -113,13 +113,13 @@ uuid = { version = "1", features = [ "serde", "v4" ] }
 
 # git
 dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main", features = [ "usdt-probes" ] }
-omicron-common = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
-nexus-client = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
-internal-dns = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
-omicron-uuid-kinds = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
+omicron-common = { git = "https://github.com/oxidecomputer/omicron", branch = "new-virtual-disk-timeseries-definition" }
```

Comment:
Note to self: Point these back to main once https://github.com/oxidecomputer/omicron/pull/6420 merges

Reply:
Done in a19a366e3afb6a49f000e022b86e2f44beac6b1a